ambari-web/app/controllers/main/host/details.js

```diff
@@ -1775,6 +1775,16 @@ App.MainHostDetailsController = Em.Controller.extend(App.SupportClientConfigsDow
         groups.push(group);
       }
     });
+    if (App.get('isHaEnabled') && installedServiceNames.contains('HDFS')) {
```

Comment:
Does HA need to be enabled to change this config?

Reply:
hi @jgolieb ,
Yes , its core-site will have parameter "ha.zookeeper.quorum" only if HA is enabled. this same file has a function : constructZookeeperConfigUrlParams 

where we have handled the condition of core-site in similar manner in the abouve function

code : 
if (App.get('isHaEnabled')) {
      urlParams.push('(type=core-site&tag=' + data.Clusters.desired_configs['core-site'].tag + ')');
    }

Reply:
Shouldn't `core-site` be updated regardless of whether HA is enabled? There could be other properties which need to change, right?

Reply:
Hi @jonathan-hurley  ,
I see the parameter ha.zookeeper.quorum in core-site only when HA is enabled. 
I tested in non-HA enabled cluster. core-site doesnt have ha.zookeeper.quorum parameter.

Reply:
Yes, but this looks like it will update all configs of type `core-site`. Shouldn't `core-site` configs always be updated?

Reply:
Hi @jgolieb ,
as Far as i test only the ha.zookpeer.quorum property of core-site will change  . the logic of updateConfigs will fetch and change only those configs defined in variable : requiredPropertiesToChange, which is in fact fetched from the UI where we have mentioned that it will be changed : 

https://github.com/apache/ambari/blob/3ad1f4a83363dd45d840d3bcd3cf0a07926d185d/ambari-web/app/controllers/main/host/details.js#L3078

i have tested in ambari-2.7.x versions its only updating ha.zookeeper.quorum value.